petastorm/reader.py

```diff
@@ -36,29 +34,99 @@
 
 logger = logging.getLogger(__name__)
 
-# For backward compatibility. Would clean this up with package reorganization in the future
-# https://github.com/uber/petastorm/issues/134
-ShuffleOptions = petastorm.shuffle_options.ShuffleOptions
-
 # Make it easier to import as "from petastorm.reader import ReaderV2"
 ReaderV2 = ReaderV2
 
 
+def make_reader(dataset_url, schema_fields=None, reader_pool=None, shuffle_row_groups=True,
```

Comment:
I thought we are going to configure parallelization with primitives (threads/process | workers_count) and hide all the workers_pool package from the user.